Software/Source/PktLoss.py

```diff
@@ -0,0 +1,68 @@
+import json
+#Global Variable
+zero=0
+one=0
+two=0
+three=0
+four=0
+zero_los=0
+one_loss=0
+three_loss=0
+four_loss=0
+#count the total number of packet types (0,1,2,3 and 4) 
+def PacketCheck(check):
+    if(check=='0'):
+        global zero

```

Comment:
Declaration of variables within the logical part is not recommended. 
Declare all variables at same location (Refer general Structure/Body of a code)